### ISSUE TITLE
Allow TV-like expansion of gallery and image parameters

### ISSUE THREAD

**Issue by drwagner13** [splittingred/Gallery#84]
Being able to add an author-credit input or other user-specified image tags and parameters would be great. This is possible in Maxi gallery and would benefit Gallery, too!

**Comment by jb2010** [comment]
I agree. This is would be a great add to Gallery. The ability for a user to add custom fields would be a great. For me this might include things like being able to add an author-credit (per the above request), source of the image (not image-path), date of the image (could be an old image, pre-EXIF), licencing/copyright etc. Having the flexibility to add user-specified fields would be fantastic!
